pkg/apis/devconsole/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -25,126 +25,177 @@ import (
-func (in *Component) DeepCopyObject() runtime.Object {
+func (in *GitSource) DeepCopyObject() runtime.Object {
 	if c := in.DeepCopy(); c != nil {
 		return c
 	}
 	return nil
 }
 
 // DeepCopyInto is an autogenerated deepcopy function, copying the receiver, writing into out. in must be non-nil.
-func (in *ComponentList) DeepCopyInto(out *ComponentList) {
+func (in *GitSourceAnalysis) DeepCopyInto(out *GitSourceAnalysis) {
 	*out = *in
 	out.TypeMeta = in.TypeMeta
-	out.ListMeta = in.ListMeta
-	if in.Items != nil {
-		in, out := &in.Items, &out.Items
-		*out = make([]Component, len(*in))
-		for i := range *in {
-			(*in)[i].DeepCopyInto(&(*out)[i])
-		}
-	}
+	in.ObjectMeta.DeepCopyInto(&out.ObjectMeta)
+	out.Spec = in.Spec
+	in.Status.DeepCopyInto(&out.Status)
 	return
 }
 
-// DeepCopy is an autogenerated deepcopy function, copying the receiver, creating a new ComponentList.
-func (in *ComponentList) DeepCopy() *ComponentList {
+// DeepCopy is an autogenerated deepcopy function, copying the receiver, creating a new GitSourceAnalysis.
+func (in *GitSourceAnalysis) DeepCopy() *GitSourceAnalysis {
 	if in == nil {
 		return nil
 	}
-	out := new(ComponentList)
+	out := new(GitSourceAnalysis)
 	in.DeepCopyInto(out)
 	return out
 }
 
 // DeepCopyObject is an autogenerated deepcopy function, copying the receiver, creating a new runtime.Object.
-func (in *ComponentList) DeepCopyObject() runtime.Object {
+func (in *GitSourceAnalysis) DeepCopyObject() runtime.Object {
 	if c := in.DeepCopy(); c != nil {
 		return c
 	}
 	return nil
 }
 
 // DeepCopyInto is an autogenerated deepcopy function, copying the receiver, writing into out. in must be non-nil.
-func (in *ComponentSpec) DeepCopyInto(out *ComponentSpec) {
+func (in *GitSourceAnalysisList) DeepCopyInto(out *GitSourceAnalysisList) {
 	*out = *in
+	out.TypeMeta = in.TypeMeta
+	out.ListMeta = in.ListMeta
+	if in.Items != nil {
+		in, out := &in.Items, &out.Items
+		*out = make([]GitSourceAnalysis, len(*in))
+		for i := range *in {
+			(*in)[i].DeepCopyInto(&(*out)[i])
+		}
+	}
 	return
 }
 
-// DeepCopy is an autogenerated deepcopy function, copying the receiver, creating a new ComponentSpec.
-func (in *ComponentSpec) DeepCopy() *ComponentSpec {
+// DeepCopy is an autogenerated deepcopy function, copying the receiver, creating a new GitSourceAnalysisList.
+func (in *GitSourceAnalysisList) DeepCopy() *GitSourceAnalysisList {
 	if in == nil {
 		return nil
 	}
-	out := new(ComponentSpec)
+	out := new(GitSourceAnalysisList)
 	in.DeepCopyInto(out)
 	return out
 }
 
+// DeepCopyObject is an autogenerated deepcopy function, copying the receiver, creating a new runtime.Object.
+func (in *GitSourceAnalysisList) DeepCopyObject() runtime.Object {
+	if c := in.DeepCopy(); c != nil {
+		return c
+	}
+	return nil
+}
+
 // DeepCopyInto is an autogenerated deepcopy function, copying the receiver, writing into out. in must be non-nil.
-func (in *ComponentStatus) DeepCopyInto(out *ComponentStatus) {
```

Comment:
Same as in the other PR. Make sure you are not deleting Component API.

Reply:
fixed in 87977b1